core-model/src/main/java/com/sequenceiq/cloudbreak/domain/Blueprint.java

```diff
@@ -58,10 +58,30 @@
     @ManyToOne
     private Workspace workspace;
 
+    private String crn;
+
+    private String creator;
+
     private String stackType;
 
     private String stackVersion;
 
+    public String getCrn() {
+        return crn;
+    }
+
+    public void setCrn(String crn) {
+        this.crn = crn;
+    }
+
+    public String getCreator() {
```

Comment:
how can we calculate the creator in case of default bleuprints?

Reply:
no creator set

Reply:
until now ;)